packages/terra-date-picker/src/DateUtil.js

```diff
@@ -28,6 +28,16 @@ class DateUtil {
 
     return momentDates.length > 0 ? momentDates : dates;
   }
+
+  // Converts date string to the ISO8601 format in UTC. If the date string is invalid and unable to convert, the originally provided string is returned.
+  static convertToISO8601(date, format) {
```

Comment:
I want it to return the original date that is passed in so that if the date is invalid, whatever date that was entered will be used as is to set as the value in the hidden input.